### ISSUE TITLE
ArrayCollection in worker

### ISSUE THREAD

**Issue by marquinalte** [doublefx/easyWorker#16]
Hi,
There any way to use an ArrayCollection in a worker?
I know that is not a good practice to do it, but I would like to pass a complex custom object to the worker and it has a lot of ArrayCollection

Thank you very much

**Comment by CatoPaus** [comment]
Yes no problem, you have to serialize, the object.. in air you have to use
the pacage from descrition file. (Or it is fixed)
18. mai 2016 4.35 p.m. skrev "marquinalte" notifications@github.com:
